### DISCUSSION TITLE
Is it possible to improve speed?

### DISCUSSION THREAD

**Discussion by whdmike** [RafaelGB/obsidian-db-folder#1067] [General]
Every time I work with DB folder table, I am frustrated about speed. As I understand, the plugin checks for information about files in a folder each time I open it and creates (completes) the table? Whenever I have more than 50 files in a folder, it takes ages. Is it not possible that it somehow remembers? I am just a user, unfortunately frustrated with Obsidian's inability to have a table of content that is stable and fast.

**Comment by LynetteCullens** [comment]
> Every time I work with DB folder table, I am frustrated about speed. As I understand, the plugin checks for information about files in a folder each time I open it and creates (completes) the table? Whenever I have more than 50 files in a folder, it takes ages. Is it not possible that it somehow remembers? I am just a user, unfortunately frustrated with Obsidian's inability to have a table of content that is stable and fast.

I've moved my vault to an SSD. It's faster, but I still consider it slow. Also, turn off any sync plugin or application that you use. It'll still be slow, but there will be a slight improvement.
